.github/workflows/ci.yml

```diff
@@ -82,6 +82,7 @@ jobs:
           - timestamps
           - output-json
           - output-junit
+          - libtest
```

Comment:
@ilslv I guess it should be named `output-libtest` to follow our conventions.

Reply:
@tyranron the reason I named it `libtest` is that in the future it shouldn't be only about outputting, but more about compatibility with all of `libtest`'s features.

Reply:
@ilslv I'm unsure it would be the case. And we always may rename it back if we will need it. But OK, let's try this way. Though, the descriptions should be adjusted a little to make this clear.